modules/mysql/main.tf

```diff
@@ -48,6 +53,7 @@ resource "google_sql_database_instance" "default" {
   region              = var.region
   encryption_key_name = var.encryption_key_name
   deletion_protection = var.deletion_protection
+  root_password       = coalesce(var.root_password, random_password.root-password.result)
```

Comment:
Looks like `root_password` was previously force new and this may result in recreation of `google_sql_database_instance` for users on the older provider. Can we default to null if no var.root_password?

https://github.com/hashicorp/terraform-provider-google/pull/13574/files